msnoise/s02new_jobs.py

```diff
@@ -93,23 +95,27 @@ def main(init=False, nocc=False):
     updated_days = []
     nfs = get_new_files(db)
     now = datetime.datetime.utcnow()
+    start_date = datetime.datetime.strptime(get_config(db, 'startdate'), '%Y-%m-%d').date()
+    end_date = datetime.datetime.strptime(get_config(db, 'enddate'), '%Y-%m-%d').date()
```

Comment:
it'd be better to get startdate & enddate from build_movstack_datelist, since there are checks for type (-negative integer or date) etc ?